### PR TITLE
Document how to connect and sign in Unity

### LINKED ISSUE #1014: Document Unity SDK connect and sign
Document using the `connectAndSign` function with the Unity SDK.

### PR DESCRIPTION
**Pull request by alexandratran**
Add how-to page for using `connectAndSign` in Unity. Wait for this feature to be released before merging. Fixes #1014.

Preview: https://docs.metamask.io/1014-unity-connect-and-sign/wallet/how-to/use-sdk/gaming/unity/connect-and-sign/

### CODE REVIEW

**Review by andreahaku — APPROVED** [review]
LGTM

File: wallet/how-to/sign-data/connect-and-sign/index.md [hunk 1]
```diff
@@ -16,17 +16,21 @@ the output of [`eth_requestAccounts`] directly to [`personal_sign`].
...
 the mobile dapp and MetaMask Mobile.
+This is useful for various purposes such as authentication and transaction verification.
 
 <p align="center">
   <video width="350" controls>
     <source src="/connect-and-sign.mp4" type="video/mp4" />
   </video>
 </p>
 
+The following instructions describe how to connect and sign in JavaScript or iOS.
```
Comment (mjsmike62) [inline wallet/how-to/sign-data/connect-and-sign/index.md:27]: instructions = procedure ?

File: wallet/how-to/sign-data/connect-and-sign/unity.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 2
+---
+
+# Connect and sign in Unity
+
+You can [connect and sign](index.md) in a single interaction from your Unity game.
+
+## Prerequisites
+
```
Comment (mjsmike62) [inline wallet/how-to/sign-data/connect-and-sign/unity.md:10]: Maybe some of these are requirements? I may be splitting hairs.

File: wallet/how-to/sign-data/connect-and-sign/unity.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- [MetaMask SDK set up](../../connect/set-up-sdk/gaming/unity.md) in your Unity game.
+
+- MetaMask Mobile version 7.10 or later.
+  Your users must have an updated version of MetaMask Mobile for this feature to work correctly.
+  For older versions of MetaMask, this function may not work as expected.
```
Comment (mjsmike62) [inline wallet/how-to/sign-data/connect-and-sign/unity.md:15]: I don't think this is a prereq? Maybe an admonition?

File: wallet/how-to/sign-data/connect-and-sign/unity.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    public void ConnectAndSign()
+    {
+        MetaMaskUnity.Instance.ConnectAndSign("This is a test message");
+    }
+    ```
+
+    Replace the test message with any string message you want to sign.
+
+    :::caution important
+    Make sure you initialize [`MetaMaskUnity.Instance`](../../../reference/sdk-unity-api.md#instance)
```
Comment (mjsmike62) [inline wallet/how-to/sign-data/connect-and-sign/unity.md:33]: Make sure = Ensure ?

File: wallet/how-to/sign-data/connect-and-sign/unity.md (hunk 2 above)
Comment (alexandratran) [reply wallet/how-to/sign-data/connect-and-sign/unity.md:10]: I'm mostly using vocabulary consistent with the rest of the MetaMask docs. We should discuss updating site-wide terms as a separate issue / a style guide update.

**Review by andreahaku — APPROVED** [review]
LGTM

**Pull request merged**